### DISCUSSION TITLE
Undo is not working on Chrome Version 111.0.5563.146

### DISCUSSION THREAD

**Discussion by visualsitemaps** [GoogleChromeLabs/ProjectVisBug#601] [Q&A]
i'm unable to get it to undo many basic edits AFTER i click away from the element.
it only works when you are inside the text element, change the text, and click undo. But if you click off it, and click undo.. nothing happens. Same for moving objects.

is there any undo/redo state management running?

**Comment by argyleink** [comment]
there currently isn't no.. it's a feature that's been for a while. the fact it works with text exiting is a browser feature, so they're the ones to blame that it works a little at all AND that it's lost after you blur the element. 

one day though we'll get an [undo/redo](#181) system in here!
